### DISCUSSION TITLE
i want to uninstall open shell but im worried my pc will crash

### DISCUSSION THREAD

**Discussion by goofyahhgoober-cmd** [Open-Shell/Open-Shell-Menu#2077] [General]
### Describe the bug

I really want to uninstall open shell but i see all the issues of openshell crashing others computers, my family will be confused of how to remove the customized start menu

### Area of issue

Installation/Other

### To reproduce

Basically how you crash it:
1. You delete or uninstall both the updater and actual app
2. you crash your computer
3. your worried you cant use your pc ever again

### Expected behavior

i expected for it to have a black screen and cursor. luckly i didnt uninstall yet

### Open-Shell version

4.4.191

### Windows version

Windows 11 24H2

**Comment by ge0rdi** [comment]
There is absolutely no need to worry.

The reports you are seeing are just rare things. You may think there is many of them because this site is meant for reporting of issues (very few people post here about good experience).

So just go ahead and uninstall it.

Installer may need to restart Explorer (if `Open-Shell` files are still in use) and that succeeds in majority of cases (I have never experienced otherwise). If Explorer is not restarted for some reason, you can always start Task Manager using `Ctrl+Shift+Esc` and start `explorer.exe` from it.

Good luck.

**Reply by goofyahhgoober-cmd** [reply]
are there also cases of BSOD? and what if `Ctrl+Shift+Esc` dosent work?

**Reply by ge0rdi** [reply]
> are there also cases of BSOD?

`Open-Shell` doesn't contain any drivers, so it can hardly cause BSOD.

> what if Ctrl+Shift+Esc dosent work?

Why shouldn't it?
In worst case just restart machine and that's it.

**Reply by Ibuprophen** [reply]
I've personally uninstalled it from a Windows 11 Laptop the other day (after reading this post) to see if there's any issues and haven't encountered any problems to this day. 

Only a few non critical files/folders and some registry entries had lingered after it was uninstalled. 

If there were issues encountered, then it would only be a result of something not related to this software. 

~Ibuprophen

**Reply by goofyahhgoober-cmd** [reply]
but what if explore.exe open everytime?

**Reply by Ibuprophen** [reply]
The issues reported regarding `explorer.exe` is specified during Install/upgrade. This wasn't (personally) experienced as an issue upon my Uninstalling it. 

~Brian Kerr

**Reply by mogekochan31** [reply]
I had the same problem, I wanted to delete openshell and return to my normal start menu, but the PC crashed, black screen, mouse was not visible, applications tab etc. was corrupted, could it have damaged the system?

**Reply by b-hayes** [reply]
I just uninstalled and ended up here because I now have black screen no start menu or task bar! But my browser was still open so I am able to search for problem and ended up here. Clearly its an issue.
PS - Installed/Uninstalled with WinGet

**Comment by steelwind345** [comment]
Just want to drop a positive comment. I was able to uninstall it today without issue. explorer.exe had to restart and so my monitors blinked, but that was it. Overall, good experiance.
